source/includes/_restApis.md

```diff
@@ -1,6 +1,6 @@
 # REST APIs (Premium)
 
-## Introduction
+## Intro
```

Comment:
Why this change?

Reply:
There is a duplicate ID selector generated by slate, which navigates to Introduction of the page instead of the Introduction of Rest API